sapientml_core/templates/model_templates/model.py.jinja

```diff
@@ -22,27 +22,45 @@ model = {{ model_name }}({{ silent }}random_state=random_state_model, {{ params
 from sklearn.multioutput import MultiOutputRegressor
 
 model = MultiOutputRegressor(model)
+
 {% elif is_multioutput_classification %}
 from sklearn.multioutput import MultiOutputClassifier
 
 model = MultiOutputClassifier(model)
 {% endif %}
+
 {% set xgbclassifier = "XGBClassifier" %}
-{% if model_name == xgbclassifier %}
-from sklearn.preprocessing import LabelEncoder
 
-label_encoder = LabelEncoder()
-target_train = pd.DataFrame(label_encoder.fit_transform(target_train), columns=TARGET_COLUMNS)
+{% if pipeline.is_multi_class_multi_targets %}
```

Comment:
I think we don't need to cosider whether the problem is multiclass or not.
This line can be `{% if is_multioutput_classification %}` . All of parts using `pipeline.is_multi_class_multi_targets` 
 in your PR can be modified. What do you think?

Reply:
Transfer `pipeline.is_multi_class_multi_targets` to `is_multioutput_classification` in [0e26e5a](https://github.com/sapientml/core/pull/59/commits/0e26e5a3e7ebc5327bbfa0ab38166e922ac93865).  
In order to make this work, `is_multioutput_classification` needs to be pass to the `_render` of `evaluation.py.jinja` in [a86d866](https://github.com/sapientml/core/pull/59/commits/a86d8664d227efd0aa9162990bc2d8226a6b80bc).

Reply:
Thank you. LGTM.